app/factories/bulkrax/with_associated_collection.rb

```diff
@@ -5,8 +5,9 @@ module WithAssociatedCollection
     # Strip out the :collection key, and add the member_of_collection_ids,
```

Comment:
is this file doing anything any more? It seems like this functionality is moved in to the object_factory and isn't included anywhere else now